DataConnectors/Gitlab/azuredeploy.json

```
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "PlaybookName": {
+            "defaultValue": "Gitlab-Sentinel",
+            "type": "string"
+        },
+        "PersonalAccessToken": {
+            "type": "securestring",
+            "defaultValue": "<Personal Access Token>",
+            "metadata": {
+                "description": "The personal access token for gitlab instance with read access."
+            }
+        },
+        "ProjectID": {
+            "type": "string",
+            "defaultValue": "<ProjectID>",
+            "metadata": {
+                "description": "The project id for gitlab projects to be audited."
+            }
+        },
+        "GroupID": {
+            "type": "string",
+            "defaultValue": "<GroupID>",
+            "metadata": {
+                "description": "The group Id for gitlab groups to be audited."
```

```suggestion
                "description": "The Group ID for the GitLab groups to be audited."
```

If this is supposed to only be a single Group ID then

```suggestion
                "description": "The Group ID for the GitLab group to be audited."
```